force-app/main/default/classes/DMLOperations.cls

```diff
@@ -180,15 +235,21 @@ public with sharing class DMLOperations {
 	 * @param contactsToUpsert List of Contacts to be upserted and linked to accounts.
 	 */
 	public static void upsertAccountsWithContacts(List<Contact> contactsToUpsert) {
-		// Loop through each Contact
-
-		// Extract the Account Name from Contact's LastName
-
-		// Get the account based on the name or create one if it doesn't exist
-
-		// Link the Contact to its Account using the Account Id
-
-		// Upsert the Contacts
+		List<Contact> upsertContactsList = new List<Contact>();
```

Comment:
Same thing here that I mentioned in a previous method. We don't always need to create a new List to upsert if the one passed in will work just as well. If that doesn't make sense, let me know!